_conferences/devfest-baltics.2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG DevFest Baltics"
```

Comment:
I meant this should be solely "GDG DevFest" and not have any additional qualifiers.